Resources/Prototypes/_Harmony/Actions/types.yml

```
@@ -10,3 +10,17 @@
       state: icon
     itemIconStyle: NoItem
     event: !type:ToggleActionEvent {}
+
+- type: entity
+  id: DividingGoose
```

Actions are usually prefixed by the word `Action`
```suggestion
  id: ActionDividingGoose
```